src/app/api/mocked/crossweb/route.ts

```diff
@@ -0,0 +1,91 @@
+import { EventsType } from '@/types/event';
+
+export const GET = async (request: Request) =>
```

Comment:
Should be available only in dev?

Reply:
We could have `.env.production`, `.env.development` etc..

Reply:
I think we will have only dev and prod

Reply:
yes